gemma/gemma_test.cc

```diff
@@ -98,7 +98,7 @@ class GemmaTest : public ::testing::Test {
   gcpp::Gemma model;
 };
 
-TEST_F(GemmaTest, Geography) {
+TEST_F(GemmaTest, DISABLED_Geography) {
```

Comment:
Should we revert these changes?

Reply:
I changed these to disabled so that ctest would skip them, since these need the external weights file. The test can still be run manually by adding --gtest_also_run_disabled_tests

Reply:
hm, not sure everyone will know that they have to run with also_run..
I understand you want to run only the cross-entropy tests sometimes. Maybe a good way forward is to move those into a separate test?